data-model/fdo-type/shared-model/0.3.0/schema/agent.json

```diff
@@ -7,61 +7,55 @@
   "properties": {
```

Comment:
Throughout the data model, we call instances of this schema: 

- dcterms:creator
- as:generator
- schema:maintainer
- ods:MergingStateChangedBy
- schema:creator
- ods:hasAgent (arrays of agent)

In my opinion, only the latter is an appropriate usage. I don't think we can use other namespaces to point to an ods object.

We can capture the specifics of an agent, such as `MergingStateChangedBy` or `creator`, using the `schema:roleName` field in this object.

Reply:
i have a few comments with this same idea but I can't find them, sorry for repeating myself here

Reply:
Ok, first thing is that we don't actually create objects of type `ods:Agent` but we create object with a couple types one being `schema:Person`. Schema is pretty open so we can use this in `schema:maintainer` and `schema:creator`. `as:generator` doesn't enforce a specific value type for the object https://www.w3.org/TR/activitystreams-vocabulary/#dfn-generator and neither does dcterms:creator (they recommend an URI, but doesn't say the value type is a string). So I actually don't think we go against any 'rules' when we do it this way.
The second point to include it all in one array and then use `schema:roleName` is a bit tricky as some of the ontologies we use do describe how the object need to look like. So the w3c web annotation model does indicate which terms need to be at what level to improve interoperability. So we can't just remove things such as dcterms:creator and as:generator from these objects.
Anyway that is how I interpreted the termsand when/what to reuse. Might be good if we could find an expert on this to have them check the model and see if we made any mistakes.

Reply:
it sounds ok to me. Otherwise we end up with having to use ods terms instead and decisions that do not improve interoperability.

Reply:
@samleeflang  can you document this discussion somewhere, maybe in release notes? Otherwise sounds great